modules/dxp/apps/segments/segments-asah-connector-test/src/testIntegration/java/com/liferay/segments/asah/connector/internal/messaging/test/IndividualSegmentsCheckerTest.java

```diff
@@ -216,8 +200,8 @@ public void testCheckIndividualSegments() throws Exception {
 						).build()));
 
 			ReflectionTestUtil.invoke(
-				_individualSegmentsChecker, "checkIndividualSegments",
-				new Class<?>[0]);
+				_individualSegmentsSchedulerJobConfiguration,
```

Comment:
Hey @regiCesar21 do not use reflection for that... just call the `getJobExecutorUnsafeRunnable` and execute the Runnable.